lighthouse-core/test/audits/seo/font-size-test.js

```diff
@@ -286,5 +289,103 @@ describe('SEO: Font size audit', () => {
         snippet: '<font size="10px">',
       });
     });
+
+    it('to external stylesheet', async () => {
```

Comment:
Why do the tests in this file start with `to...`?  Makes them opaque imo.  I can't really tell what they are testing just by reading the description.

```suggestion
    it('links to external stylesheet', async () => {
```

Reply:
because it's a continuation of the `describe('attributes source location')`.

Reply:
![image](https://user-images.githubusercontent.com/4071474/61750785-1bb53380-ad5b-11e9-9391-f2ae89e2da04.png)

Reply:
Hm, I don't like this.  But I guess the prior art is there, and makes some sort of sense.